marconi-chain-index/src/Marconi/ChainIndex/Indexers/Utxo.hs

```diff
@@ -20,9 +20,9 @@
 -- + table: unspent_transactions
 --
 -- @
---      |---------+------+-------+-----------+-------+-------+------------------+--------------+------|
---      | Address | TxId | TxIx  | DatumHash | Datum | Value | InlineScriptHash | InlineScript | Slot |
---      |---------+------+-------+-----------+-------+-------+------------------+--------------+------|
+--      |---------+------+-------+-----------+-------+-------+------------------+--------------+------+-----------+------------|
+--      | Address | TxId | TxIx  | DatumHash | Datum | Value | InlineScriptHash | InlineScript | Slot | BlockHash | BlockIndex |
```

Comment:
I would rename `BlockIndex` to `TxIndexInBlock`. `BlockIndex` might confuse us with the index of the block in the blockchain.

Reply:
e8987b660, it's called block index in dbsync, but our name is clearer :)